datafusion/sqllogictest/test_files/select.slt

```diff
@@ -1225,6 +1225,63 @@ statement ok
 SELECT * EXCEPT(a, b, c, d)
 FROM table1
 
+# try zero column with LIMIT, 1 row but empty
+statement ok
```

Comment:
I had these tests checked out locally to double check they actually did produce no columns but a single row and by using `query` rather than `statement`

Thing looks great to me, so I pushed a commit to this branch to update the tests

Reply:
🤔  that seems to cause errors on mac, so I reverted the change